daffodil-runtime1/src/test/scala/org/apache/daffodil/processors/input/TestICU.scala

```diff
@@ -149,4 +151,32 @@ class TestICU {
     //assertEquals(1234L, num)
     //assertEquals(7, pp.getIndex)
   }
+
+  // Shows that even if a decimal format pattern doesn't contain a decimal, the
```

Comment:
Should say "doesn't contain a decimal point, ..." (the point is, it's is missing 'point')